includes/objectcache/MemcachedClient.php

```diff
@@ -1157,6 +1157,18 @@ function _set( $cmd, $key, $val, $exp ) {
 				$flags |= self::COMPRESSED;
 			}
 		}
+		// Wikia change - begin - @author: wladek
+		// Log details if we try to store value that will not fit the default item_max_size
+		if ( $len > 1024 * 1024 - 2 ) {
+			// default item_max_size is 1mb, 2 characters are reserved for trailing "\r\n"
+			if ( class_exists( 'Wikia\\Logger\\WikiaLogger' ) ) {
+				\Wikia\Logger\WikiaLogger::instance()->debug( 'MemcachedClient: large value' , [
```

Comment:
In #6246 I've added `Loggable` trait to this class, so you can now call `$this->error()`.